benchmark.go

```diff
@@ -132,6 +137,9 @@ func runBenchmark(out output, allOpts Options, m benchmarkMethod) {
 		states[i] = newBenchmarkState(statter)
 	}
 
+	progressBar := pb.StartNew(opts.MaxRequests)
```

Comment:
There's 2 main types of benchmarks:
 - by total number of requests
 - by total duration

we're only taking total number of requests into account but we should also think about duration.

we could progress bars for both if both are specified?